example/lib/controls.js

```diff
@@ -0,0 +1,51 @@
+const allKeys = Array.prototype.slice.call(
```

Comment:
See [the docs](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from)